### DISCUSSION TITLE
v1.8.0

### DISCUSSION THREAD

**Discussion by cody-scott** [dbt-msft/dbt-sqlserver#544] [General]
## What's Changed
* Fix for adding columns to snapshots by @ericmuijsvanoord in https://github.com/dbt-msft/dbt-sqlserver/pull/477
* DBT 1.8 and test coverage updates by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/513
* PR for 1.8.0rc1 by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/515
* Update drop all schemas to target only schemas of a table in a given index by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/520
* updated behaviour of custom tests using with statements by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/518
* added test to validate swapping from table to view and view to table by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/521
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/526
* added incremental fix and validation test by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/531
* update application name to app by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/533
* docs(readme): switch to english url for odbc driver by @hongbo-miao in https://github.com/dbt-msft/dbt-sqlserver/pull/453
* readded batch logic and included a test by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/532
* 1.8.0rc2 release by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/534
* fixed logic for snapshot and added tests by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/537
* Bump black from 24.2.0 to 24.8.0 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/529
* Bump actions/setup-python from 4 to 5 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/525
* Bump docker/login-action from 2.1.0 to 3.3.0 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/524
* Bump mypy from 1.8.0 to 1.11.2 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/527
* 540 units tests not working as expected by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/541
* 1.8.0 Release PR by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/543

## New Contributors
* @ericmuijsvanoord made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/477
* @hongbo-miao made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/453

**Full Changelog**: https://github.com/dbt-msft/dbt-sqlserver/compare/v1.7.4...v1.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dbt-msft/dbt-sqlserver/releases/tag/v1.8.0'>v1.8.0</a>.</em>

**Comment by cody-scott** [comment]
For future releases, the major and minor versions will align with their respective DBT versions, but we will increment patch versions as new updates and releases are added here.

ie 1.8.X will be tied to the 1.8 releases, 1.9.X to 1.9, etc.

**Reply by axellpadilla** [reply]
Thanks @cody-scott !

**Comment by cody-scott** [comment]
1.8.1 was released to address a problem with databases and non-standard names.
